tests/test-icon.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Checks if icon data matches a subject icon.
+ * @param {import('../index').SimpleIcon} icon Icon data
+ * @param {import('../index').SimpleIcon} subject Icon to check against icon data
+ * @param {String} slug Icon data slug
+ */
+const testIcon = (icon, subject, slug) => {
+  describe(icon.title, () => {
+    it('has the correct "title"', () => {
+      expect(subject.title).toStrictEqual(icon.title);
+    });
+
+    it('has the correct "slug"', () => {
+      expect(subject.slug).toStrictEqual(slug);
```

Comment:
There might be a good reason for this, but why not:

```suggestion
      expect(subject.slug).toStrictEqual(icon.slug);
```

Reply:
`icon.slug` won't be defined in all cases though. `icon` is directly from `_data/simple-icons.json`

Reply:
Ah indeed, my bad :sweat_smile: Thank you for pointing that out

(I knew there was a reason but couldn't think of it, felt it was better to comment than to leave it)